apps/managedidentity/docs/RUN_IMDS.md

```diff
@@ -0,0 +1,162 @@
+# Running the IMDS Source
```

Comment:
Rename to "msi_manual_test.md" ? It won't be just IMDS.

Reply:
Changed